mflowgen/full_chip/tile_array/get_tile_array_outputs.sh

```diff
@@ -19,13 +16,8 @@ else
 fi
 mkdir -p outputs
 
-if [ "$WHICH_SOC" == "amber" ]; then
-    cp -L *cadence-genus-genlib/outputs/design.lib outputs/tile_array_tt.lib
-    cp -L *synopsys-dc-lib2db/outputs/design.db outputs/tile_array_tt.db
-else
     cp -L *synopsys-ptpx-genlibdb/outputs/design.db outputs/tile_array_tt.db
     cp -L *synopsys-ptpx-genlibdb/outputs/design.lib outputs/tile_array_tt.lib
```

Comment:
let's get rid of these indents now that the if/else is gone

Reply:
done.